src/docs/faq.md

```diff
@@ -1,88 +1,101 @@
-* Please see [here](https://github.com/WeirdTreeThing/chromebook-linux-audio)
+### Will the audio driver still work when re-installing Windows?
+* Yes, on the device the license was created. If the internal drive was replaced, see below.
 
-#### How do I get my top row keys on Linux to act like how they did on chromeOS?
-* Follow the instructions [here](https://github.com/WeirdTreeThing/cros-keyboard-map).
+### I upgraded my Chromebooks' internal drive and my audio/Thunderbolt 4 license is not working, what do I do?
+* The license is tied to the serial number of your internal drive. **You do not need to buy another license**. Send CoolStar a message through Patreon, and wait for about a week.
 
-#### I just want to boot Linux from USB on my Chromebook, what do I need to do?
-* Check the Supported Devices page, to ensure your device has functional RW_LEGACY firmware available. If so, flash RWL (RW_LEGACY firmware) and use that.
+### How do I buy Windows audio drivers in bulk?
+* A discount is only available for Chromebooks containing an `Intel 6th gen processor (up to Core i5)`, `N3350`, `N4000`, or, `N4020` processor, otherwise you must pay full price. You must purchase 10 or more licenses to be eligible for a discount. If you are eligible for a discount, you should ask in the [Chrultrabook Forum](https://forum.chrultrabook.com).
 
-#### I just installed the UEFI firmware, and now my device boots to a black screen that says 'shell' - what do I do?
-* You're in the EFI shell; the firmware boots there when it can't find a valid UEFI boot device, either externally (USB, SD) or internally. Just type `exit` to get back to the the UEFI settings menu.
+### Can I use one driver license across multiple devices?
+* No. One license per device.
 
-#### Should I install 32 or 64-bit OS?
-* The UEFI Full ROM firmware only supports 64-bit OS, so be sure to use the 'x86_64' version of whatever OS you want to install. USB media created from 32-bit ISOs will simply fail to boot.
+<br>
 
-#### I'm using your UEFI firmware, installed my OS [Linux], and it still boots to the EFI shell - what do I do?
-* Sounds like your Linux distro doesn't install the EFI bootloader in the default location - no worries, it's an easy fix:
-  * Type `exit` to return to the UEFI settings menu, then select Boot Maintenance Manager. From there, select Boot From File, then navigate to and boot from `/EFI/[distro name]/grubx64.efi` (where [distro name] will be fedora, arch, debian, etc). Once your OS is booted, open a terminal/shell, and type the following (observing case):
+## Linux Questions
 
-   * `sudo su`
-   * `mkdir -p /boot/efi/EFI/BOOT`
-   * `cp /boot/efi/EFI/[distro name from above]/grubx64.efi /boot/efi/EFI/BOOT/BOOTX64.efi`
+### How come audio isn't working on Ubuntu / Ubuntu forks?
+* Ubuntu and Ubuntu-based distros may have issues and are not supported, try switching to another distro. Debian is a great alternative if you are used to ubuntu.
```

Comment:
"Debian is a great alternative if you are used to **u**buntu."
"Ubuntu" needs to be capitalized